### DISCUSSION TITLE
797

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#973] [Comments]
# 797

6월 1일에 결혼식을 했다.수백명의 지인들 앞에서 향해 축가를 불렀다.혹시나 못할 수도 있으니, 와이프를 비롯해 축사를 해주는 친구에게도, 축가를 불러주는 친구에게도, 그 외 주변의 모든 이에게 이야기 하지 않고 몰래 준비했다.노래에 자신이 없다보니 계속 연습해도 마음에 들지 않으면 안해야겠다는 생각도 있었다.아무도 모르면 아무일도 아니다.4, 5월 2달간 주 1회씩 진행하다가 결혼식 2주 전부터는 주 2회씩 숨고를 통해 집 근처에서 보컬 트레이닝 하시는 선생님께 배웠다.처음 보는 사람 (선생님) 앞에서 반주 없이, 마이크 없이 그냥 노래를 부르는 일은 최근 들어 가장 부끄러운 순간이였다.'나'의 노래를 직접 듣는게 처음이였고, 그렇게 부른 노래를 녹음하고 처음 들었을때는 1절만 듣고 끌 정도로 참기 힘..

https://jojoldu.tistory.com/797

**Comment by lsb530** [comment]
아이고...ㅠㅠ
미천하지만 제가 노래를 조금 해서, 미리 알았더라면 조금의 피드백을 드렸을 것 같은데 아쉽네요
2022, 2023, 2024년의 동욱님이 인프콘에서 발표하시는 모습을 보니까
더욱 더 말씀도 잘하시고, 창피함도 덜 하신것같아보이던데요!
응원합니다

**Comment by eunsu1014** [comment]
일본 호스트바 업계 1위인 롤랜드가 이렇게 말했습니다.
"쪽팔리거나 부끄러운 감정을 좋아한다. 그럴 때 마다 성장할 수 있기 때문이다."
성장하기 위해서는 부끄러움을 감수해야만 하는 것 같습니다 ㅎㅎ
좋은 글 잘 읽고 갑니다!

**Comment by alex-gibeom** [comment]
그간 열심히 눈팅만 해오던 동욱님 블로그에 처음 댓글을 달아보네요
제가 첫 컨퍼런스 발표 준비를 할 때, 동욱님이 멘토링을 해주시면서 말씀해 주신 문장들이 문득문득 기억나곤 합니다 😄
“기범님 너무 긴장될 때는 이렇게 생각해 보시는 게 어떨까요?”
“내가 하고 싶은 이야기를 누군가의 방해 없이 많은 사람들이 집중해서 들어준다는 게 얼마나 설레고 좋은 일인가요. 이 흔치 않은 기회를 재밌게 즐겨봐요”

동욱님 덕분에 긴장되지만, 창피하지만, 망할 것 같지만, 그래도 일단 해보자는 마음으로 무사히 컨퍼런스를 잘 끝냈었네요
익숙하지 않은 컨퍼런스 발표 경험이 막상 잘 끝내고 나보니 너무 좋은 에피소드로 남아있습니다 😊
저도 앞으로 동욱님처럼 1년에 최소 한 번 이상은 새로운 시도를 이것저것 해봐야겠어요ㅎㅎ 🙌

---

결혼식에서 아무도 예상하지 못한 신랑 축가가 시작되고, “와 내가 향로 노래를 다 들어보네”라는 하객들 반응이 들렸던 게 생각나네요 :)
긴장 가득한 상태로 열심히 부르시는 그날의 동욱님 모습은 정말 누구보다도 행복해 보이고 멋있어 보였습니다 👍👍
항상 행복하십쇼!!

**Comment by jws1837** [comment]
죄송하지만 혹시 녹음본이 있을까요? 들어보고 싶네요 😆

**Comment by mthekid** [comment]
결혼 축하드립니다~

**Comment by gustjtmd** [comment]
저도 한달전에 결혼해서 축가를 직접 불렀는데 향로님도 그러셨다니까 신기하네요
